README.md

```diff
@@ -72,11 +72,11 @@ We try to make sure that all issues in the issue tracker are as close to fully s
     **Note:** You can see the emulator setup used on CI in  `.circleci/config.yml`.
 
 ## Testing a form without a server
-When you first run Collect, it is set to download forms from [https://opendatakit.appspot.com/](https://opendatakit.appspot.com/), the demo server. You can sometimes verify your changes with those forms but it can also be helpful to put a specific test form on your device. Here are some options for that:
```

Comment:
@seadowg I won't be setting up a new appspot instance so this should stay the stay the same for now. 

@ln Maybe we can setup demo.central.getodk.org and get rid of all the appspot references? This should ideally be paired with a change in the defaults of Collect.

Reply:
Whooops. Sorry I should have checked that one.